types/inputmask/index.d.ts

```diff
@@ -513,6 +513,13 @@ declare namespace Inputmask {
          * Format of the unmasked value. This is only effective when used with the datetime alias.
          */
         outputFormat?: string;
+
+        /**
+         * Add new definitions to this inputmask.
+         */
+        definitions?: {
+          [key: string]: Definition;
```

Comment:
The indentation seems wrong. Please make sure 4 spaces are used for indentation to keep the code consistent with the rest of the repo. See [read me](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).